deploy/buildtools/buildconfig.py

```diff
@@ -71,6 +71,12 @@ def __init__(self,
                 self.s3_bucketname = aws_resource_names_dict['s3bucketname']
         self.post_build_hook = recipe_config_dict['post_build_hook']
 
+        self.fpga_bit_builder_dispatcher_class_name = recipe_config_dict.get('fpga-platform', "F1BitBuilder")
```

Comment:
None of the recipes changed to add this. I would either 1. add it to each recipe or 2. for now just set it to "F1BitBuilder" by default. We can modify this later.